app/readme.md

```diff
@@ -11,9 +11,6 @@ To **run the app locally**, you need to do **four things**:
 1. Get the code, navigate to the `app` folder and install `docker` and `docker-compose`
 2. Compile the protocol buffers
 3. Launch the backend with `docker-compose`
-4. Install and launch the web frontend with `yarn`
```

Comment:
I would keep this step, but instead include a link to the front-end repo, e.g.:
Install and launch the [web frontend](https://github.com/Couchers-org/web-frontend)

Reply:
right!